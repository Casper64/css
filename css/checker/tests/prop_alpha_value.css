
.alpha {
    opacity: 0.5;
    opacity: 50%;
}

.invalid_alpha {
    opacity: -2;
    opacity: 2;
    opacity: 110%;
    opacity: -20%;
}