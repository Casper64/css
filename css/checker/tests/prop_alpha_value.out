css/checker/tests/prop_alpha_value.css:8:14: error: Alpha value must be a percentage between 0-100% or between 0 and 1
    6 | 
    7 | .invalid_alpha {
    8 |     opacity: -2;
      |              ~~~
    9 |     opacity: 2;
   10 |     opacity: 110%;
css/checker/tests/prop_alpha_value.css:9:14: error: Alpha value must be a percentage between 0-100% or between 0 and 1
    7 | .invalid_alpha {
    8 |     opacity: -2;
    9 |     opacity: 2;
      |              ~~
   10 |     opacity: 110%;
   11 |     opacity: -20%;
css/checker/tests/prop_alpha_value.css:10:14: error: Alpha value must be a percentage between 0-100% or between 0 and 1
    8 |     opacity: -2;
    9 |     opacity: 2;
   10 |     opacity: 110%;
      |              ~~~~~
   11 |     opacity: -20%;
   12 | }
css/checker/tests/prop_alpha_value.css:11:14: error: Alpha value must be a percentage between 0-100% or between 0 and 1
    9 |     opacity: 2;
   10 |     opacity: 110%;
   11 |     opacity: -20%;
      |              ~~~~~
   12 | }